recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,19 @@
+osx_impl:  # [osx]
+  - nomps  # [osx]
+  - mps    # [osx]
+
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
+  - 11.0                   # [osx and arm64]
+  - 10.9                   # [osx and x86_64]
+  - 12.3                   # [osx]
+MACOSX_SDK_VERSION:        # [osx]
```

Comment:
it may be that we don't need to specify this.

Reply:
No, it is working as desired

Reply:
the current problem is with my logic selectors in the names 😵‍💫

Reply:
I think I know what you mean tho, the 11.0 and 10.9 are the default values, but how can we zip them together without being explicit?

Reply:
I have a feeling that `MACOSX_SDK_VERSION` will be specified to the other defined if not specified.

Reply:
maybe.

Reply:
will test locally and report back

Reply:
```
  zip fields in zip_key group frozenset({'osx_impl', 'MACOSX_DEPLOYMENT_TARGET', 'MACOSX_SDK_VERSION'}) are not all the same length

```

Reply:
remove `MACOSX_SDK_VERSION` from the zip keys below as well.